features/guests_added.feature

```diff
@@ -18,7 +18,7 @@ Feature: See if you are attending an event
       | lastname     | Guest          |
       | email        | test@guest.com |
     Then I should see "You have successfully registered!"
-    And I should see "Test Guest"
+    And I should not see "Test Guest"
```

Comment:
I think it might be better to remove the last check entirely, or find a better way to test it.